BuildTools/CMake/src/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+
+# target
+add_executable( example-cmake main.cpp )
+
+# include path for example target
+target_include_directories( example-cmake PRIVATE ../../../common )
```

Comment:
You use so little from common files that I'd be sorely tempted to write change main.cpp to avoid this dependency entirely. There are going to be people who think this is necessary, otherwise.

It does mean introducing a bit of duplication, but I think that's probably ok for examples, if it makes them simple and independent.

Reply:
An include path shouldn't confuse and may want to move to more shared use later.